packages/pages/src/util/runtime.ts

```diff
@@ -32,8 +39,39 @@ class Runtime {
       throw new Error("Not running in Node.");
     }
 
+    if (!this.version) {
+      throw new Error("No version found.");
+    }
+
     return +this.version.split(".")[0];
   }
+
+  isBrowser(): boolean {
+    return (
+      typeof window != "undefined" &&
+      typeof window.document !== "undefined" &&
+      !("Deno" in window)
```

Comment:
Maybe the final check should be `&& !isDeno() && !isNode()`

Reply:
Removed `isBrowser` check because it will be the default case since we want to confirm at least one environment is set